### PR TITLE
document working k8s version

### LINKED ISSUE #1054: MountVolume.SetUp failed for volume "cert" : secret "kfserving-webhook-server-cert" not found
/kind bug

**What steps did you take and what happened:**
I am running `minikube` as described in the documentation here: https://github.com/kubeflow/kfserving#install-kfserving-in-5-minutes-on-your-local-machine

```bash
./hack/quick_install.sh
```

Then Check KFServing controller installation, 

```
kubectl get po -n kfserving-system
NAME                             READY   STATUS             RESTARTS   AGE
kfserving-controller-manager-0   1/2     CrashLoopBackOff   5          5m2s
```

When i look in the dashboard I see the following message, 

```
MountVolume.SetUp failed for volume "cert" : secret "kfserving-webhook-server-cert" not found
```

**What did you expect to happen:**

```
kubectl get po -n kfserving-system
NAME                             READY   STATUS    RESTARTS   AGE
kfserving-controller-manager-0   2/2     Running   2          13m
```

**Environment:**

see `./hack/quick_install.sh`

- Minikube version: v1.9.2
- Kubernetes version: (use `kubectl version`): 1.18
- OS (e.g. from `/etc/os-release`): macos

### PR DESCRIPTION
**Pull request by riklopfer**
**What this PR does / why we need it**:
Try to ensure a working version of Kubernetes. 

**Which issue(s) this PR fixes**:
Fixes #1054 
Probably #1003 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by riklopfer** [comment]
@animeshsingh hopefully this is an easy review

File: hack/quick_install.sh [hunk 1]
```diff
@@ -72,7 +72,15 @@ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/relea
 kubectl wait --for=condition=available --timeout=600s deployment/cert-manager-webhook -n cert-manager
 cd ..
 # Install KFServing
-kubectl apply -f install/${KFSERVING_VERSION}/kfserving.yaml --validate=false
+K8S_MINOR=$(kubectl version | perl -ne 'print $1."\n" if /Server Version:.*?Minor:"(\d+)"/')
+if [[ $K8S_MINOR -gt 17 ]]; then
+  echo "Kubernetes minor version must be < 18 got ${K8S_MINOR}"
```
Comment (animeshsingh) [inline hack/quick_install.sh:77]:
```suggestion
  echo "Kubernetes minor version must be <= 17 got ${K8S_MINOR}"
```

Comment (animeshsingh) [reply hack/quick_install.sh:77]: other than that, 

/lgtm

**Pull request merged**